py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -330,8 +330,13 @@ def determine_resources(ncameras, jobdesc, queue, nexps=1, forced_runtime=None,
     if jobdesc in ('ARC', 'TESTARC'):
         ncores          = 20 * (10*(ncameras+1)//20) # lowest multiple of 20 exceeding 10 per camera
         ncores, runtime = ncores + 1, 45             # + 1 for worflow.schedule scheduler proc
-    elif jobdesc in ('FLAT', 'TESTFLAT'):
+    elif jobdesc in ('FLAT', 'TESTFLAT', 'TILENIGHT'):
         ncores, runtime = 20 * nspectro, 25
+        if system_name[0:10] == 'perlmutter':
+            # ~150 frames per node hour plus
+            # 20-minute overhead for a single node
+            # perlmutter-[cpu,gpu] have timefactor=[0.7,0.5]
+            runtime = 20 + int(60. / 140. * ncameras * nexps)
```

Comment:
Originally the intention was that we would request the same number of cores across any system for any given step, and use `timefactor` to adjust the runtime needed to achieve that step on that system.  Here you have a system-specific runtime calculation separate from `timefactor`, and it appears that the resulting resources requested for TILENIGHT on cori would just be wrong (runtime=25 min).

If TILENIGHT won't work on cori, let's be explicit about refusing to schedule it, or otherwise have scheduling logic that works across all systems, leveraging `timefactor` as much as possible to encode the system differences in the yaml file rather than in the code.

Reply:
Done. @sbailey please have a look and see if this works.